modules/widgets/simple-payments.php

```diff
@@ -13,6 +13,33 @@
 	 * Display a Simple Payment Button as a Widget.
 	 */
 	class Jetpack_Simple_Payments_Widget extends WP_Widget {
+		private static $currencie_symbols = array(
```

Comment:
Typo here. 

Also, how will we keep this in sync? Is this also hardcoded in the SP endpoint?

Reply:
Okay so we're probably basing this off of https://github.com/Automattic/wp-calypso/blob/f2494ca3b040fb7f87f9f5b0e49975a237d27150/client/components/tinymce/plugins/simple-payments/dialog/form.jsx#L41-L67 ?

For a start let's also link back to https://developer.paypal.com/docs/integration/direct/rest/currency-codes/

I'd also be surprised if WordPress didn't already have a currency to symbol mapping someplace.

This may be easier to maintain as an endpoint, but we can leave that for a future update.

Reply:
For WordPress.com we have a static function on the store that provides currencies, but there's no similar class on Jetpack 🙁 

Typo fixed and docs link added on 66c5e612a3725afd2f672cdba15f8d4cbceb1386.